packages/babylon/src/tokenizer/index.js

```diff
@@ -363,7 +368,7 @@ export default class Tokenizer extends LocationParser {
 
         default:
           if (
-            (ch > 8 && ch < 14) ||
+            (ch > charCodes.backSpace && ch < charCodes.shiftOut) ||
             (ch >= 5760 && nonASCIIwhitespace.test(String.fromCharCode(ch)))
```

Comment:
`5760` can be converted to a constant?

Reply:
Yes, I have it here https://github.com/xtuc/charcodes/blob/master/packages/charcodes/src/index.js#L103. But it's kind of a strange char. I would prefer to use a function which tests against a range. What do you think?

Reply:
I like the idea 👍

Reply:
what would be the function's name though 🤣

Reply:
good point :joy: no idea

Reply:
`charcodes.isUnusualWhiteSpace`? :thinking: :joy:

Reply:
We could make it handle also the `charCodes.space` and `charCodes.nonBreakingSpace` and call it `charCodes.isSpaceSeparator` (https://www.compart.com/en/unicode/category/Zs)